### PR TITLE
Sorting results in module username_checker by the platform's siterank

### PR DESCRIPTION
**Pull request by krinistof**
1. Removed unnecessary duplicates from `username_checker.json`
2. Implemented sorting for output
3. Made crawling instructions in `username_checker.json` sorted by site rank, for better maintainability

### CODE REVIEW

**Comment by saeeddhqan** [comment]
Recheck the module and let me know.

File: modules/osint/username_search.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 def thread(self, data, query,thread_count):
 	threadpool = concurrent.futures.ThreadPoolExecutor(max_workers=thread_count)
-	futures = (threadpool.submit(check, self, data[site]['url'].format(self.urlib(query).quote_plus(query).replace('.','%2E')), site, data) for site in data)
+	futures = (threadpool.submit(check, self, data[site]['url'].format(self.urlib(query).quote_plus.replace('.','%2E')), site, data) for site in data)
```
Comment (krinistof) [inline modules/osint/username_search.py:40]: `quote_plus` isn't defined in `core/urlib.py`, so it makes error. Made a corresponding method, might need revision

**Comment by krinistof** [comment]
Module checked, and works. Feel free to verify.

File: core/util/urlib.py [hunk 2]
```diff
@@ -65,9 +65,13 @@ def check_urlfile(self, file):
 	def quote(self):
 		if '%' not in self.url:
 			self.url = urlparse.quote(self.url)
-			return self.url
-		else:
-			return self.url
+		return self.url
+
+	@property
+	def quote_plus(self):
```
Comment (saeeddhqan) [inline core/util/urlib.py:71]: urlparse.quote_plus

File: modules/osint/username_search.py [hunk 3]
```diff
@@ -70,12 +69,65 @@ def module_api(self):
...
-	self.save_gather(output, 'osint/username_search', query, output=self.options.get('output'))
+	self.save_gather(
+		output,
+		'osint/username_search',
+		query,
+		output=self.options.get('output')
+	)
+
+    
+	output['links'] = sorted(
```
Comment (saeeddhqan) [inline modules/osint/username_search.py:80]: does this section work correctly?

Comment (krinistof) [reply modules/osint/username_search.py:80]: As far as I can tell, it works, yes.

**Comment by saeeddhqan** [comment]
Shall we?

**Pull request merged**